### DISCUSSION TITLE
Have not supported php 8.2 yet

### DISCUSSION THREAD

**Discussion by qiulang** [facade/ignition#468] [General]
I find "facade/ignition" have not support 8.2. When my composer.json look like these,

    "require": {
        "php": "^8.2",
        "clue/socket-raw": "^1.3",
        "fideloper/proxy": "^4.4",
        "fruitcake/laravel-cors": "^2.0",
        "gregwar/captcha": "1.*",
        "guzzlehttp/guzzle": "^7.0.1",
        "itsgoingd/clockwork": "^5.1",
        "jacobcyl/ali-oss-storage": "^2.0",
        "james-heinrich/getid3": "^1.9",
        "laravel/framework": "^9.33",
        "laravel/horizon": "^5.7",
        "laravel/tinker": "^2.5",
        "php-curl-class/php-curl-class": "^8.6",
    },
    "require-dev": {
        "facade/ignition": "^2.17",
        "phpunit/phpunit": "^9.5.28"
    },
    ...

Run `composer install --no-scripts --no-autoloader --no-dev` will complain

`laravel/framework[v9.43.0, ..., 9.x-dev] require dragonmantank/cron-expression ^3.3.2 -> found dragonmantank/cron-expression[dev-set-up-semaphore, dev-update-v3-php8, dev-master, dev-issue/35-fix-and-or-problem, dev-allow-phpstan-failures, v1.0.0, ..., v1.2.1, v2.0.0, ..., v2.3.1.x-dev, v3.0.0, ..., v3.3.1] but it does not match the constraint.`

Only after I remove "facade/ignition" from "require-dev", composer (2.5.1) work fines

**Comment by rubenvanassche** [comment]
Please use spatie/laravel-ignition, this package is abandoned.
